freva-rest/src/freva_rest/api_config.toml

```diff
@@ -112,3 +112,102 @@ client_id = "freva"
 # Secret key associated with the client ID. Keep this secure as it is used to
 # authenticate the API to the OIDC provider.
 client_secret = ""
+
+
+### IMPORTANT: If you uncomment the whole configuration below, you can use 
+## the secondary-backend for connecting to all kind of Rational Database
+## Management System (RDBMS) backend and search engine (SE) backend like
+## Elasticsearch or OpenSearch. You can uncomment one of the following
+## sections based on the backend you want to connect to.
+
+# # PostgreSQL Configuration
+
+# [secondary-backend]
```

Comment:
Why did you call this secondary backend? What's the primary. Also, why is this commented? 

Why you don't just call it `search-backend` or something else and merge this section with the solr section?